idd/Energy+.idd.in

```diff
@@ -72866,6 +72876,87 @@ DemandManager:Thermostats,
        \note can be selected by entering <Zone Name><space><Global Thermostat Object Name>.
        \type object-list
        \object-list ZoneControlThermostaticNames
+       
+  DemandManager:Ventilation,
+       \memo used for demand limiting Controller:OutdoorAir objects.
+       \extensible:1 Duplicate the last field Controller:OutdoorAir Name
+  A1 , \field Name
+       \required-field
+       \type alpha
+       \reference DemandManagerNames
+  A2 , \field Availability Schedule Name
+       \note Availability schedule name for this DR. Schedule value > 0 means the DR is available.
+       \note If this field is blank, the DR is always available.
+       \type object-list
+       \object-list ScheduleNames
+  A3 , \field Limit Control
+       \required-field
+       \type choice
+       \key Off
+       \key FixedRate
+       \key ReductionRatio
+  N1 , \field Minimum Limit Duration
+       \type integer
+       \minimum> 0
+       \units minutes
+       \note If blank, duration defaults to the timestep
+  N2 , \field Fixed Rate
+       \memo Used in case when Limit strategy is set to FixedRate
```

Comment:
Should use the real field name here.  "Used when Limit Control is set to FixedRate"